<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWork</class>
 <widget class="QMainWindow" name="AddWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить произведение</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Icons.qrc">
    <normaloff>:/Icons/MainIcon/Resources/MainIcons/free-icon-book-4341050.png</normaloff>:/Icons/MainIcon/Resources/MainIcons/free-icon-book-4341050.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_edit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Номер издания</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edit_num">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Тип</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type_box"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Авторы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="author_box"/>
        </item>
        <item>
         <widget class="QTableWidget" name="author_tab"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="add_author">
            <property name="text">
             <string>Добавить автора</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="del_author">
            <property name="text">
             <string>Удалить автора</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="auth_del_num">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Жанры</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="genre_box"/>
        </item>
        <item>
         <widget class="QTableWidget" name="genre_tab"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="add_genre">
            <property name="text">
             <string>Добавить жанр</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="del_genre">
            <property name="text">
             <string>Удалить жанр</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="gen_del_num">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="add_work">
      <property name="text">
       <string>Добавить произведение</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
